x/fundraising/spec/01_concepts.md

```diff
@@ -4,18 +4,71 @@
 
 ## Fundraising Module
 
-The `x/fundraising` Cosmos SDK module provides an oppotunity for new projects to onboard the ecosystem. It does not only allow projects to raise funds, but also increase their brand awareness before launching their projects.
+The `x/fundraising` Cosmos SDK module is a module to raise funds as an auction of tokens. This fundraising module provides an opportunity for a new project to onboard the ecosystem. It does not only allow the project to raise funds, but also increase its brand awareness before launching the project.
 
 ## Auction Types
 
-There are two different types of auctions in the fundraising module.
+This fundraising module provides two different types of auctions: 1) Fixed Price Auction and 2) Batch Auction.
 
-### Fixed Price Auction
+## Fixed Price Auction
 
-A fixed price auction type is the most basic type of an auction that is first come first served way to raise funds. When an autioneer creates this fixed price auction, they must determine the fixed start price that is proportional to the paying coin denom. Once it is created, bidders can only bid with the start price of the auction. For example, an auctioneer sets 0.5 as start price for X coin and the paying coin is Y coin. How many X coins does a bidder receives if they bids 10Y coin? It is calculated as Y coin over start price (10/0.5), which results to 20X coin.  
+This fixed price auction is to sell a given amount of tokens on a first-come, first-served basis.
 
-### English Auction
+### What an auctioneer does:
+When an auctioneer creates a fixed price auction, it must determine the following parameters.
 
-An english auction type is an ascending dynamic auction that an auctioneer decides the starting price of the selling amount of coin and bidders bid to purchase the amounts of the coin. Competition for the price, not how fast bidders can bid.
+- **Selling Token**: the denom of tokens to be auctioned,
+- **Paying Token**: the denom of tokens to be used for payment,
+- **Price**: fixed amount of the paying tokens to get a selling token (i.e., amount of paying tokens per selling token),
+- **Auction Start Time**: when the auction starts,
+- **Auction End Time**: when the auction ends,
+- **Offering Quantity**: total amount of selling tokens to be auctioned.
 
-## Extended Auction Round(s)
+### What bidders can/cannot do:
```

Comment:
Since we use a singular for an auctioneer above `### What an auctioneer does`, I think it is better to make it consistent. What do you think?
```suggestion
### What a bidder can/cannot do:
```

Reply:
It looks better. Thanks.